main.go

```diff
@@ -58,3 +59,22 @@ func postTasks(c *gin.Context) {
 	tasks = append(tasks, newTask)
 	c.IndentedJSON(http.StatusCreated, newTask)
 }
+
+func putTasks(c *gin.Context) {
+	var newTask task
+	id := c.Param("id")
```

Comment:
can we define types in Go?

Reply:
c.Param("id") can only be a string since that's how it's declared in the task structure, but yes, I've changed it so that the type is defined in putTasks as well.